va-gov/pages/index.md

```diff
@@ -5,12 +5,118 @@ title: Home
+    url: /disability/
+    hub: disability
+    description: File a claim for disability compensation for conditions related to your military service, and manage your benefits over time.
+  - heading: Education and Training
+    url: /education/
+    hub: education
+    end_row: true
+    description: Apply for and manage your GI Bill and other education benefits to help pay for college and training programs.
+  - heading: Careers and Employment
+    url: /careers-employment/
+    hub: careers
+    description: Apply for vocational rehabilitation services, get support for your Veteran-owned small business, and access other career resources.
+  - heading: Pension
+    url: /pension/
+    hub: pension
+    description: Apply for monthly payments for wartime Veterans and survivors with limited or no income who meet certain age and disability requirements.
+  - heading: Housing Assistance
+    url: /housing-assistance/
+    hub: housing
+    end_row: true
+    description: Find out if you're eligible for a VA-backed home loan. If you have a service-connected disability, see if you qualify for a housing grant to help you live more independently.
+  - heading: Life Insurance
+    url: /life-insurance/
+    hub: life-insurance
+    description: Explore VA life insurance options for Veterans, Servicemembers, and families. Manage your policy online, file claims for benefits, and access helpful resources.
+  - heading: Burials and Memorials
+    url: /burials-memorials/
+    hub: burials
+    description: Get help planning a burial in a VA national cemetery, order a headstone or other memorial item to honor a Veteran's service, and apply for survivor and dependent benefits.
+  - heading: Records
+    url: /records/
+    hub: records
+    description: Apply for a printed Veteran ID card, get your VA benefit letters and medical records, and learn how to apply for a discharge upgrade.
+news:
+  - img: /img/homepage/home-1.jpg
+    alt: Four Veterans standing together in front of the United States flag.
+    title: Make the Connection Website
+    href: https://maketheconnection.net/
+    description: Hear Veterans share their stories of recovery, and find resources near you.
+  - img: /img/homepage/home-2.jpg
+    alt: Prescription medicine bottles being processed through the automated system at the VA Mail Order Pharmacy.
+    title: Health Care Access and Quality
+    href: https://www.accesstocare.va.gov/
+    description: Get wait times and compare quality and patient-satisfaction data for VA health facilities.
+  - img: /img/homepage/home-3.jpg
+    alt: Close-up of an American flag patch on the sleeve of a U.S. military combat uniform.
+    title: VA Rapid Appeals
+    href: https://www.blogs.va.gov/VAntage/48141/vas-rapid-appeals-modernization-program-ramp-now-open-appeals/
+    description: Find out how to get a faster decision on a pending disability compensation appeal through the VA Rapid Appeals Modernization Program (RAMP).
+
+
```

Comment:
🔥 🔥 🔥 🔥 🔥 🔥

Reply:
This looks great @jbalboni --no edits from me.